<ui version="4.0" >
 <comment> * OpenRPT report writer and rendering engine
 * Copyright (C) 2001-2012 by OpenMFG, LLC
 *
 * This library is free software; you can redistribute it and/or
 * modify it under the terms of the GNU Lesser General Public
 * License as published by the Free Software Foundation; either
 * version 2.1 of the License, or (at your option) any later version.
 *
 * This library is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
 * Lesser General Public License for more details.
 *
 * You should have received a copy of the GNU Lesser General Public
 * License along with this library; if not, write to the Free Software
 * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 * Please contact info@openmfg.com with any questions on this license.</comment>
 <class>ImportWindow</class>
 <widget class="QMainWindow" name="ImportWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Report Import Tool</string>
  </property>
  <widget class="QWidget" name="widget" >
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <widget class="QGroupBox" name="groupBox1" >
      <property name="title" >
       <string>Reports</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QListWidget" name="_reports" >
         <property name="selectionMode" >
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="_add" >
           <property name="text" >
            <string>&amp;Add</string>
           </property>
           <property name="shortcut" >
            <string>Alt+A</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="_remove" >
           <property name="text" >
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line1" >
           <property name="frameShape" >
            <enum>QFrame::HLine</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Sunken</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="_import" >
           <property name="text" >
            <string>&amp;Import</string>
           </property>
           <property name="shortcut" >
            <string>Alt+I</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="_selectAll" >
           <property name="text" >
            <string>Select All</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox2" >
      <property name="title" >
       <string>Messages</string>
      </property>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>11</number>
       </property>
       <item>
        <widget class="QTextEdit" name="_log" >
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="unnamed" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="fileOpenAction" />
    <addaction name="separator" />
    <addaction name="fileExitAction" />
   </widget>
   <widget class="QMenu" name="helpMenu" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="helpContentsAction" />
    <addaction name="helpIndexAction" />
    <addaction name="separator" />
    <addaction name="helpAboutAction" />
   </widget>
   <addaction name="unnamed" />
   <addaction name="helpMenu" />
  </widget>
  <action name="helpContentsAction" >
   <property name="text" >
    <string>&amp;Contents...</string>
   </property>
   <property name="iconText" >
    <string>Contents</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="helpIndexAction" >
   <property name="text" >
    <string>&amp;Index...</string>
   </property>
   <property name="iconText" >
    <string>Index</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="helpAboutAction" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
   <property name="iconText" >
    <string>About</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="fileExitAction" >
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
   <property name="iconText" >
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="fileOpenAction" >
   <property name="text" >
    <string>&amp;Add</string>
   </property>
   <property name="iconText" >
    <string>&amp;Add</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>_reports</tabstop>
  <tabstop>_add</tabstop>
  <tabstop>_remove</tabstop>
  <tabstop>_import</tabstop>
  <tabstop>_selectAll</tabstop>
  <tabstop>_log</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_selectAll</sender>
   <signal>clicked()</signal>
   <receiver>_reports</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>526</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel" >
     <x>422</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
